Change the call + FAQs (#595)
ropensci/roweb3: content/champions/howtoapply.md

Reviewer: Where is this file linked from? It seems to have the same information as in the /champions/_index.md...

+++

## Champions

> You need to fill out an application form. [This template shows the information we will require](/champions/files/champions_template). The application forms have to be completed _in English_.

Reviewer: ```suggestion
Please fill out an application form. [This template shows the information we will require](/champions/files/champions_template). Applications need to be submitted _in English_.
```
